app/utils/wellknown.js

```diff
@@ -0,0 +1,110 @@
+// thanks to npm well-known-wallets-hns 
+import https from "https";
+import hdns from "hdns";
+
+const init = hsd => hdns.setServers(['103.196.38.38', '103.196.38.39', '103.196.38.40'] );
```

Comment:
Can we use the local `hsd` instead?

Reply:
I'm not sure . And if use local hsd the resolve mode must be opened.

Reply:
Agreed. And keep these as fallback as there's a setting to disable hsd's DNS servers now (so it doesn't clash with hnsd, etc.) and is disabled by default.

Reply:
And what is important the hdns request just send an https request to the hdns servers. Which our local hsd is hard to catch that request.

Reply:
Even as fallback, this should be moved to a place where users can override/update (settings page).

Reply:
I'm happy to see that hdns.io supports DNSSEC now! (when did that start?) However I agree a local user-secured option should be available. User could already have HNS set up on their network or computer (even Fingertip). Maybe we can look into some way to bundle the DNSSEC chain into the TLS handshake the way that CA chain is passed in SSL

Reply:
For a sec I was excited and tried it out -_+
They'll (all public resolvers) also need to consider how to communicate the public key that should be checked by clients. Using the hard-coded key would be :facepalm:

Reply:
@Falci I tried the dns servers bob-wallet started. But seems nslookup get nothing.  Maybe i can make hdns server settings to be used to validate.

Now we need two configurations. 
1. wellknown wallet address validate strategy . 
2.  hdns server used to vallidate tlsa record.